site/assets/scss/_clipboard-js.scss

```diff
@@ -17,16 +17,63 @@
   }
 }
 
-// Boosted mod
 .btn-clipboard,
 .btn-edit {
-  position: absolute;
-  right: 0;
-  z-index: 10;
-  @include border-radius();
+  display: block;
+  padding: .5em;
+  line-height: 1;
+  // Boosted mod: no specific color
+  background-color: $gray-100;
+  border: 0;
+  @include border-radius(.25rem);
+
+  ////////////////////////////////////////////////////////
+  ////////////////////////////////////////////////////////
+  ////////////////////////////////////////////////////////
+  // TODO: see if we can't use a btn-* class instead
```

Comment:
We could indeed use `.btn .btn-no-outline` but it introduces a border on `:hover` and with a `.border-0` -> not the same behavior

Reply:
Another solution could be to use the `.btn-check` combined with the `border: 0` to have the expected result

Reply:
Another solution could be to use the `.btn-check` combined with the `border: 0` to have the expected result